packages/allow-scripts/README.md

```diff
@@ -2,20 +2,27 @@
 
 A tool for running only the dependency lifecycle hooks specified in an _allowlist_.
 
+> For an overview of LavaMoat tools see [the main README](https://github.com/LavaMoat/LavaMoat/tree/main/README.md)
+
+
 ### Install
 
 Adds the package to start using it in your project. be sure to include the `@lavamoat/` namespace in the package name
 
 ```sh
 yarn add -D @lavamoat/allow-scripts
+npm i -D @lavamoat/allow-scripts
```

Comment:
Maybe write this as 

```
`yarn add -D @lavamoat/allow-scripts`

or

`npm i -D @lavamoat/allow-scripts`
```

So that it doesn't look like you have to run both of these commands?

Reply:
I'll fix that and see how annoyed we are with the visual clutter :P